hardware/esp8266com/esp8266/libraries/ESP8266WiFi/src/ESP8266WiFi.h

```diff
@@ -113,7 +113,7 @@ class ESP8266WiFiClass
       *
       * return: one value of wl_status_t enum
       */
-    int softAPdisconnect(bool wifioff = false);
```

Comment:
Changing this default value will likely change behaviour of existing sketches. Is this change really required?